var/spack/repos/builtin/packages/py-multiqc/package.py

```diff
@@ -43,8 +43,17 @@ class PyMultiqc(PythonPackage):
     depends_on('py-jinja2@2.9:', type=('build', 'run'))
     depends_on('py-lzstring', type=('build', 'run'))
     depends_on('py-future@0.14.1:', type=('build', 'run'))
-    depends_on('py-spectra', type=('build', 'run'))
+    depends_on('py-spectra@0.0.10:', type=('build', 'run'))
     depends_on('py-matplotlib', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
     depends_on('py-pyyaml', type=('build', 'run'))
     depends_on('py-simplejson', type=('build', 'run'))
+
+    # I don't see these dependencies in setup.py or anywhere
+    # in the multiqc source, but they end up being imported
+    # e.g.
+    # requests in lib/python2.7/site-packages/multiqc/utils/megaqc.py
```

Comment:
What do you mean by you do not see this anywhere in the sources if requests is imported there?

Reply:
Yeah, ignore that. I wasn't paying close enough attention to which version I was on. Fixed.